people_tool/static/people_tool/js/controllers/SearchController.js

```diff
@@ -98,9 +98,23 @@
             // can still display a clickable anchor link
             var lastName = (full.name_last.trim().length > 0) ?
                 full.name_last : '(none)';
-            return '<a href="' + '#/people/' + full.univ_id + '/courses/">'
+            return '<a href="' + '#/people/' + full.univ_id + '/courses/" ' +
+                'ng-click="setSelectedPersonInfo(\''+full.name_last+'\',\''+
+                full.name_first+'\','+full.univ_id+',\''+full.email_address+'\''+',\''+full.role_type_cd+'\')">'
```

Comment:
I was looking at how to do this in angular and it looks like this is a place where we could use a service to pass the data from one controller to another. Now that we have a working version with this code, wanna try creating a service? We could do some XP coding?